### DISCUSSION TITLE
expense frontend error while creating logs in elastic

### DISCUSSION THREAD

**Discussion by devilathag** [learndevopsonline/learndevopsonline#1496] [Q&A] [answered]
Hi sir,

i have terminated all old instances and creating new instance run and install same commands as it is but frontend ip was net working IP:  34.228.144.89

<img width="943" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/144150880/081de428-a4cf-4e63-8247-bd1b425ccae2">

**Comment by sreedharm07** [comment]
Hi @devilathag 
did still problem exists? 
if yes can you please turn on your machines and can you share workstation ip

**Comment by devilathag** [comment]
Dear Sir,

Please find the workstation ip : 3.80.183.26 and do the needful


Best regards
G V Devilatha

On Mon, Oct 2, 2023 at 10:32 AM sreedharm07 ***@***.***>
wrote:

> Hi @devilathag <https://github.com/devilathag>
> did still problem exists?
> if yes can you please turn on your machines and can you share workstation
> ip
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/learndevopsonline/learndevopsonline/discussions/1496#discussioncomment-7161095>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BCLZCYC5ZB4XFW7ITHQMOHLX5JDHDANCNFSM6AAAAAA5NIOYPU>
> .
> You are receiving this because you were mentioned.Message ID:
> <learndevopsonline/learndevopsonline/repo-discussions/1496/comments/7161095
> @github.com>
>


-- 
Respected Sir/Madam,

          I would like to apply for the Opportunities at your firm. I am
attaching my Curriculum Vitae. Iam G. DeviLatha, B.Tech in the stream of
“Electronic and Communication”, passed out in 2016 with an aggregate of
72%.  I am sincere and eager to learn new skills have good communication
skills and can work effectively in a team. I do believe my strengths would
definitely add values to your reputed company, given an opportunity to
associate myself with your leading corporate. I look forward to hearing
from you.



Thanks & Regards,

G. Devi Latha

**Reply by sreedharm07** [reply, marked as answer]
Hi @devilathag
your ansible directory structure was wrong. task directory was missing.
app_prereq.yml is supposed to be in **common/tasks/app_prereq** but in your code prereq file is in path - **common/app_prereq**
please place app_prereq file under tasks directory.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/bf8e0292-50d5-47b0-96e1-035a8b721720)
